cmd/boulder-va/main.go

```diff
@@ -80,6 +81,17 @@ func main() {
 		err = rpc.NewValidationAuthorityServer(vas, vai)
 		cmd.FailOnError(err, "Unable to setup VA RPC server")
 
+		if c.VA.GRPC != nil {
+			s, l, err := bgrpc.NewServer(c.VA.GRPC)
+			cmd.FailOnError(err, "Unable to setup VA gRPC server")
+			err = wrappers.RegisterValidationAuthorityGRPCServer(s, vai)
+			cmd.FailOnError(err, "Unable to register VA gRPC server")
+			go func() {
```

Comment:
Instead of running either AMQP or gRPC and AMQP I think we should only be running one or the other. (If we need to support both we can run two binaries which will make debugging where messages are coming from much simpler)